lib/consts/Types.ts

```diff
@@ -4,3 +4,8 @@ export type Error = {
 };
 
 export type ScriptElement = Buffer | number;
+
+export enum ChainTypes {
```

Comment:
Please create an own file `ChainType.ts` in this folder. Also please use the singular form of the noun for naming `enum`s.